frontend/src/components/tracks/expectations/_styles.css

```diff
@@ -0,0 +1,35 @@
+.wwe {
```

Comment:
What does `wwe` stand for?

Reply:
https://giphy.com/gifs/wwe-wrestling-summerslam-26Ff17mcKrYRvLnOM 

`expectations` would be better I guess? :D

Reply:
Yeah `expectations` would be much clearer. 👍